Release notes/v11.5.x (#685)
mia-platform/documentation: docs/release-notes/v11.5.5.md

---
id: v11.5.5
title: Version 11.5.5 Release Notes
sidebar_label: v11.5.5
image: "img/release-note-link-preview.png"
---

_Oct 16th, 2023_

## Console

### Customized branding in the console 

> From now on it will be possible to set your custom brand in the Console, so that it replaces Mia-Platform's one both within the Console and in the login pages. To enable this customization, please contact your product contact person.

Reviewer: ```suggestion
From now on it will be possible to set your custom brand in the Console, so that it replaces Mia-Platform logo one both within the Console and in the login pages. To enable this customization, please contact your product contact person.

:::info
This feature is only available for self-hosted installations
:::
```

---
id: v11.5.5
title: Version 11.5.5 Release Notes
sidebar_label: v11.5.5
image: "img/release-note-link-preview.png"
---

_Oct 16th, 2023_

## Console

> ### Customized branding in the console 

Reviewer 2: ```suggestion
### Customized branding in the Console 
```

From now on it will be possible to set your custom brand in the Console, so that it replaces Mia-Platform logo one both within the Console and in the login pages. To enable this customization, please contact your product contact person.

:::info

> This feature is only available for self-hosted installations

Reviewer: ```suggestion
This feature is only available for self-hosted installations.
```

:::

## How to update your Console

> For on-premise installations, please contact your Mia-Platform referent and upgrade your _Console Helm Chart_ to `v9.5.5-hotfix.2`.

Reviewer: because we cannot increment 9.5.6 version without conflicting with other releases
